.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on:
+  - pull_request
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest requests_mock
+      - name: Test with pytest
+        run: |
+          pytest
```

Comment:
can we use `make test` here? We could also implement the test target in the `green-db/Makefile` similar to, e.g., `workers-test-deploy` target.